### PR TITLE
Added CamOps Fatigue Attribute to Crew

### PR DESCRIPTION
**Pull request by IllianiCBT**
- Updated code to include a CamOps "fatigue" attribute for crew members, reflecting their state of tiredness
- Modified messages and dialog fields to reflect new CamOps fatigue attribute
- Adjusted indentation and formatting in various places for code cleanliness
- Removed redundancy in four places
- Updated unit test "CrewTest" to reflect changes
- Renamed method "setFatigue()" to "setCrewFatigue()" in "Crew.java" to differentiate from mhq's fatigue system
- Modified "MULParser.java" to handle parsing fatigue details from a .mul file

This PR is required for https://github.com/MegaMek/mekhq/pull/4101 and https://github.com/MegaMek/mekhq/pull/4116

### CODE REVIEW

File: megamek/src/megamek/client/ui/swing/CustomPilotView.java [hunk 1]
```diff
@@ -428,6 +437,10 @@
         return Integer.parseInt(fldTough.getText());
     }
 
+    public int getCrewFatigue() {
+        return Integer.parseInt(fldFatigue.getText());
```
[the remark this answers is not in the document]
Comment (IllianiCBT) [reply megamek/src/megamek/client/ui/swing/CustomPilotView.java:441]: False positive

**Review by Sleet01 — APPROVED** [review]
Looks good, but please consider whether we might benefit from making the old crew fatigue getter/setter functions call the new versions at index 0.

File: megamek/src/megamek/common/Crew.java [hunk 2]
```diff
@@ -1093,12 +1103,12 @@ public void setToughness(int t, int pos) {
         toughness[pos] = t;
     }
 
-    public int getFatigue() {
-        return fatiguePoints;
+    public int getCrewFatigue(int pos) {
```
[the remark this answers is not in the document]
Comment (IllianiCBT) [reply megamek/src/megamek/common/Crew.java:1106]: That's a great idea. I'll see it done, tomorrow

**Comment by IllianiCBT** [comment]
> Looks good, but please consider whether we might benefit from making the old crew fatigue getter/setter functions call the new versions at index 0.

Restored the requested getter/setter and marked as Depreciated. Didn't find any uses, but they're restored. :)

**Pull request merged**